### ISSUE TITLE
Detail List - Set column ColumnActionsMode.disabled, but still focus-able in scan mode of Narrator

### ISSUE THREAD

**Issue by MdAmeerMind** [OfficeDev/office-ui-fabric-react#11944]
Hi,
We have requirement like have column without header name so it does not have header it will not accessible/ focusable using tab. To achieve this I have implemented column with `columnActionsMode: ColumnActionsMode.disabled`. 
It works fine With NVDA and Window Narator normally, but when I am using Windows Narrator scan mode, Column is  focus able.

Repro : https://codepen.io/mdameer/pen/YzXwKvx?editors=0010

Step:
1) Open above link in Edge
2) Start Windows Narrator
3) Press keys `CapsLock+SpaceBar`
4) Press key `T` , So focus will move to entire grid table.
5) Press Keys `Ctrl+Alt+RightArrow`

Now observe that focus is move to Non interactive column

Expected Behavior:
Focus will no go on Column without name.

**Comment by xugao** [comment]
@MdAmeerMind - `ColumnActionsMode.disabled` makes the column to be non-focusable. scan mode is never designed to skip non-focusable nodes. So i would say this is by design. could you point to me any documentation that says otherwise?

**Comment by MdAmeerMind** [comment]
Hi @xugao , Thanks your input. If any document available will be helpful.

**Comment by xugao** [comment]
@MdAmeerMind  - no problem. Sorry, we in general lack accessibility spec for our component, which is something we actively trying to improve.

**Comment by MdAmeerMind** [comment]
Hi @xugao , can you please consider this as bug. Accessibility team is not accepting it as design.
